packages/spacecat-shared-http-utils/src/auth/handlers/scoped-api-key.js

```diff
@@ -46,6 +46,7 @@ export default class ScopedApiKeyHandler extends AbstractHandler {
       this.log(`No API key entity found in the data layer for the provided API key: ${apiKeyFromHeader}`, 'error');
       return null;
     }
+    this.log(`API Key entity: ${apiKeyEntity}`, 'debug');
```

Comment:
Should this object be stringified?

Reply:
Removing these logs as we might not need them anymore.